app/assets/javascripts/components/client_comments.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * @prop comments - comments associated with current client
+ * @prop client - current client
+ * @prop user - current user
+ */
+
+class ClientComments extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: []
+    };
+  }
+
+  componentDidMount() {
+		Requester.get(`/api/comments/client/${this.props.client.id}`).then((comments) => {
+			this.setState({ comments });
+		});
+	}
+
+  addComment(val) {
+    const newComment = {content: val,
+                        client_id: this.props.client.id,
+                        thread_id: 1,
+                        user_id: this.props.user.id
+                        }
+    this.state.comments.push(newComment);
```

Comment:
One potential issue with this is that we might be pushing comments into our state before we're sure that they are actually saved into the db.